services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDeviceRegistryUtils.java

```diff
@@ -35,6 +35,14 @@
  */
 public final class MongoDbDeviceRegistryUtils {
 
+    /**
+     * The name of the JSON property containing the device identifier.
+     */
+    public static final String FIELD_DEVICE = "device";
+    /**
+     * The name of the JSON property containing the data.
+     */
+    public static final String FIELD_DATA = "data";
```

Comment:
IMHO we don't need this as it seems to be a re-declaration of `RegistrationConstants.FIELD_DATA`, isn't it?